### PR TITLE
deprecation-message-additions

### PR DESCRIPTION
**Pull request by daskairo**
## Changes

I added some deprecation messages to the "Roblox Globals" Documentation for some methods that have been replaced. This is to add further context as to why they are replaced, and to further direct readers to the newer methods.

I have none, its mostly for convinience to the reader.

## Checks

By submitting your pull request for review, you agree to the following:

- [X] This contribution was created in whole or in part by me, and I have the right to submit it under the terms of this repository's open source licenses.
- [X] I understand and agree that this contribution and a record of it are public, maintained indefinitely, and may be redistributed under the terms of this repository's open source licenses.
- [X] To the best of my knowledge, all proposed changes are accurate.

### CODE REVIEW

**Comment by daskairo** [comment]
I think I made a few mistakes before sending the request.

Changes were made to:
- delay()
- spawn()
- wait()

Documentation Link: https://create.roblox.com/docs/reference/engine/globals/RobloxGlobals

File: content/en-us/reference/engine/globals/RobloxGlobals.yaml [hunk 1]
```diff
@@ -101,6 +101,9 @@ functions:
         summary: ''
     tags:
       - Deprecated
+    deprecation_message: |
+      This method has been superceded by `Library.task.delay()` and should not
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/globals/RobloxGlobals.yaml:105]:
```suggestion
      This method has been superseded by `Library.task.delay()` and should not
```

File: content/en-us/reference/engine/globals/RobloxGlobals.yaml [hunk 2]
```diff
@@ -268,6 +271,9 @@ functions:
         summary: ''
     tags:
       - Deprecated
+    deprecation_message: |
+      This method has been superceded by `Library.task.spawn()` and should not
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/globals/RobloxGlobals.yaml:275]:
```suggestion
      This method has been superseded by `Library.task.spawn()` and should not
```

File: content/en-us/reference/engine/globals/RobloxGlobals.yaml [hunk 3]
```diff
@@ -391,6 +397,9 @@ functions:
         summary: ''
     tags:
       - Deprecated
+    deprecation_message: |
+      This method has been superceded by `Library.task.wait()` and should not
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/globals/RobloxGlobals.yaml:401]:
```suggestion
      This method has been superseded by `Library.task.wait()` and should not
```

File: content/en-us/reference/engine/globals/RobloxGlobals.yaml (hunk 1 above)
Comment (IgnisRBX) [reply content/en-us/reference/engine/globals/RobloxGlobals.yaml:105]: Just changing these to the correct spelling (for whatever reason in English, "supercede" becomes "superseded".

**Review by IgnisRBX — APPROVED** [review]

**Pull request merged**